webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -158,7 +160,7 @@ function GithubItems(props) {
                     style={style.subtitle}
                 >
                     {item.created_at && ('Opened ' + formatTimeSince(item.created_at) + ' ago')}
-                    {userName && ' by ' + userName}
+                    {userName && <a href={userProfile}> by {userName}</a>}
```

Comment:
The linter is complaining about the JSX/string formatting here:

`163:56  error  Missing JSX expression container around literal string: “by”  react/jsx-no-literals`

Maybe try something like this:
```suggestion
                    {userName && (
                        <>
                            {' by '}
                            <a href={userProfile}>{userName}</a>
                        </>
                    )}
```

Reply:
Once I get my environment setup, I'll go ahead and test out this change.